app/client/views/homes/assignManger/assignManager.js

```diff
@@ -0,0 +1,41 @@
+import assignManagerSchema from "./schema";
+
+Template.assignManager.onCreated(function() {
+  const templateInstance = this;
+  templateInstance.subscribe("allUsers");
+  templateInstance.currentManager = new ReactiveVar(null);
+  Meteor.call("getCurrentManager", templateInstance.data.groupId, function(
+    err,
+    userEmail
+  ) {
+    if (err) return;
+    else {
+      templateInstance.currentManager.set(userEmail);
+    }
+  });
+});
+
+Template.assignManager.helpers({
+  assignManagerSchema() {
```

Comment:
I believe this should just return the schema, imported at the top of the file. Fetching the `users` can be done in the schema file directly, to keep logic together for easy understanding.